packages/console/src/Console.tsx

```diff
@@ -401,6 +401,7 @@ export class Console extends PureComponent<ConsoleProps, ConsoleState> {
       return;
     }
 
+    this.scrollConsoleHistoryToBottom();
```

Comment:
This will always scroll to the bottom after any command run. It should only scroll if it was already at the bottom I think. Unless we want to change that behavior, but would need Don to approve a new behavior

Reply:
Actually it looks like we already scroll to bottom when a command starts, so this is probably fine